platform/linuxbsd/display_server_x11.h

```diff
@@ -362,6 +362,9 @@ class DisplayServerX11 : public DisplayServer {
 	virtual void window_set_ime_active(const bool p_active, WindowID p_window = MAIN_WINDOW_ID);
 	virtual void window_set_ime_position(const Point2i &p_pos, WindowID p_window = MAIN_WINDOW_ID);
 
+	virtual void window_set_vsync_mode(DisplayServer::VSyncMode p_vsync_mode, WindowID p_window = MAIN_WINDOW_ID) override;
+	virtual DisplayServer::VSyncMode window_get_vsync_mode(WindowID p_vsync_mode) const override;
```

Comment:
This class didn't have `override` before this, so now Clang complain about the others that are missing.

Reply:
Is the use of `override` desired? If so, I could add it where it should be (only display_server_iphone and display_server_osx use it currently), otherwise I could open a PR to remove the keyword from all display_server classes except osx and iphone.

Reply:
> Is the use of `override` desired?

Yes, it is. I'm not sure why it wasn't used in those classes before.

Reply:
I opened a PR for this: #50513